e2e/ctl_v3_member_test.go

```diff
@@ -101,3 +105,85 @@ func ctlV3MemberRemove(cx ctlCtx, memberID, clusterID string) error {
 	cmdArgs := append(cx.PrefixArgs(), "member", "remove", memberID)
 	return spawnWithExpect(cmdArgs, fmt.Sprintf("%s removed from cluster %s", memberID, clusterID))
 }
+
+func memberAddTest(cx ctlCtx) {
+	// by this, one member has started
+	mresp1, err := getMemberList(cx)
+	if err != nil {
+		cx.t.Fatal(err)
+	}
+
+	var (
+		memberToAdd    = "newmember"
+		dataDirPath    string
+		clientURLToAdd = url.URL{Scheme: "http", Host: fmt.Sprintf("localhost:%d", etcdProcessBasePort+10)}
+		peerURLToAdd   = url.URL{Scheme: "http", Host: fmt.Sprintf("localhost:%d", etcdProcessBasePort+11)}
+	)
+	cmdArgs := append(cx.PrefixArgs(), "member", "add", memberToAdd, fmt.Sprintf("--peerURLs=%s", peerURLToAdd.String()))
+	if err = spawnWithExpect(cmdArgs, " added to cluster "); err != nil {
+		cx.t.Fatal(err)
+	}
+	// by this, another member has been added
+
+	// start the added node
+	dataDirPath, err = ioutil.TempDir("", memberToAdd+".etcd")
+	if err != nil {
+		cx.t.Fatal(err)
+	}
+	defer os.RemoveAll(dataDirPath)
+	args := []string{
+		"--name", memberToAdd,
+		"--listen-client-urls", clientURLToAdd.String(),
+		"--advertise-client-urls", clientURLToAdd.String(),
+		"--listen-peer-urls", peerURLToAdd.String(),
+		"--initial-advertise-peer-urls", peerURLToAdd.String(),
+		"--initial-cluster-token", cx.cfg.initialToken,
+		"--data-dir", dataDirPath,
+		"--initial-cluster", fmt.Sprintf("%s=%s,%s=%s", mresp1.Members[0].Name, mresp1.Members[0].PeerURLs[0], memberToAdd, peerURLToAdd.String()),
+		"--initial-cluster-state", "existing",
+	}
+	proc, err := spawnCmd(append([]string{"../bin/etcd"}, args...))
```

Comment:
I think it's enough to just get the memberlist and confirm the peer address is there-- starting an etcd process for the new member seems like too much